pkg/kubelet/volumemanager/cache/desired_state_of_world.go

```diff
@@ -213,9 +213,9 @@ func (dsw *desiredStateOfWorld) AddPodToVolume(
 		volumeName = volumehelper.GetUniqueVolumeNameForNonAttachableVolume(podName, volumePlugin, volumeSpec)
 	}
 
-	volumeObj, volumeExists := dsw.volumesToMount[volumeName]
+	_, volumeExists := dsw.volumesToMount[volumeName]
```

Comment:
since `volumeExists` is only used in the if statement, it would make sense to put this statement in the if statement:
`if _, volumeExists := ...; volumeExists {`